spec/models/user_spec.rb

```diff
@@ -85,7 +85,7 @@
     end
 
     context 'when no organisations are returned' do
-      let(:ids) { [User::STUDENT_WITHOUT_ORGANISATIONS_ID] }
+      let(:ids) { [SecureRandom.uuid] }
```

Comment:
Minor: It would be nice to have a one-line explanation in the commit note as to *why* you can now safely do this.

Reply:
The same applies to the other commits where you've removed the other similar constants, but I haven't added a comment for each.

Reply:
I've added the following to each of the relevant commits:

> Now that we can stub Hydra's UserInfo API and our UserInfo API to respond with users containing arbitrary user IDs we're no longer restricted to using these constants.